pkg/postage/postagecontract/mock/contract.go

```diff
@@ -60,3 +65,9 @@ func WithDiluteBatchFunc(f func(ctx context.Context, batchID []byte, newDepth ui
 		m.diluteBatch = f
 	}
 }
+
+func WithExpiresBatchesFunc(f func(ctx context.Context) error) Option {
```

Comment:
This function is unused.

Reply:
the contract was completed and approved yesterday so this part is now a wip